### DISCUSSION TITLE
Apologies and reassurances

### DISCUSSION THREAD

**Discussion by johnbent** [tekinged/missing#455] [Announcements]
All, 

I know our open issues are currently numbering 193. There are probably three types of open issues.

1. Still legitimately open waiting for resolution. But resolution won't come unless someone aggressively keeps commenting in them to try to get us resolved. I'm also in Palau now and am meeting with PLC tomorrow. I'm planning to ask them for a weekly meeting where they can help with these.
2.  Have sufficient info and just need one of us to write an AUTOPARSE for them. 
3. Have sufficient info and just need one of us to make some simple changes in the database that don't require an AUTOPARSE or that AUTOPARSE doesn't yet know how to handle.

So, apologies I haven't done more.

Also, reassurances that the work is still valuable even though most of them haven't been completed in many months. One, I will eventually complete many, maybe most, maybe all of them. Two, even if some are never completed, they are still available and linked off tekinged so they are preserved forever. Maybe we'll all die before they're complete and @Lukes-Imeyuns's ghost will be horrified to see a future AI complete this work for us. :)

As always, thanks for being a part of one of my very favorite communities!

**Comment by johnbent** [comment]
Oh, here's a way you all can help. Especially you Palauans. Please reach out to other native speakers and request they come watch this repo and get involved. The more folks commenting, the more likely we can complete issues.

**Comment by Lukes-Imeyuns** [comment]
Oh gosh, let us humxns do the work, not the robot. 🤣 Jk!

John, when is the event again? I have a couple of folks in mind to
encourage to attend.

Trying to find the email with the info but...

Thank you!

Ps. Dererengum el kuk mo mesang a Imeyuns. :)

**Comment by johnbent** [comment]
Hey Jelga,

What event do you mean? :)

I do have a meeting with PLC later today but I actually tried to invite
some others and they said they want the first one with just me.

Do you mean the general solicitation for more volunteers? For that, there
is no specific event. People can just show up whenever they want and start
helping.

We could certainly schedule some events! We could do a
weekly/biweekly/monthly/quarterly zoom call and go through the Issues in
focused group sessions. We could also do a kickoff event for new volunteers
to explain how things work. Let me know if you think one or both of these
would be useful and if you can help set them up please.

In the meantime, people can just show up and help. Here are the
instructions for it:

We have a self-appointed committee of volunteers. We post questions about
potentially missing and incorrectly defined words, discuss them, and then I
push consensus changes back into the database. We need more volunteers! If
you're willing to try it out, please do the following:
1. go to https://github.com/tekinged/missing
2. In the upper-right of the page, either sign in or sign up depending if
you already have a github account you want to use for this
3. Once you are signed in, then please click on the "Watch" button [mine
says 'Unwatch' because I've already watched]
image.png
4. Select All activity

It is only a few emails a week. If you try it and get too many emails, you
can always unwatch or I can suggest some ways to reduce the amount you get.

No worries if you aren't interested. I'd much prefer you spend your
tekinged volunteer uploading audio by the way! But that's my fault . . . .

There is also the Dekaingeseu tab with more ways to help. The best one
there is audio recordings but that's currently broken and I haven't found
the time to fix it yet.

Probably I should make a page with this info to help more people find out
about this.

Thanks!

John
